code/game/objects/items/melee/fireaxe.dm

```diff
@@ -52,3 +52,32 @@
 /obj/item/melee/axe/scrap/ComponentInitialize()
 	. = ..()
 	AddComponent(/datum/component/two_handed, force_unwielded=5, force_wielded=22, icon_wielded="[base_icon_state]_w")
+
+/*
+Blunt
+ */
+/obj/item/melee/axe/sledgehammer //add wall + structure damage later once Gristlebee's pr is merged
+	icon_state = "sledgehammer"
+	base_icon_state = "sledgehammer"
+	name = "breaching sledgehammer"
+	desc = "A large hammer used by the Gorlex Marauder splinters. As powerful as a weapon as it is a shipbreaking and mining tool."
+	force = 5
+	armour_penetration = 40
+	attack_verb = list("bashed", "smashed", "crushed", "smacked")
+	hitsound = list('sound/weapons/genhit1.ogg', 'sound/weapons/genhit2.ogg', 'sound/weapons/genhit3.ogg')
+	slot_flags = NONE
+	sharpness = IS_BLUNT
+	tool_behaviour = TOOL_MINING
```

Comment:
i'd make it so this is added when wielded rather than being a constant

Reply:
Forgot to do this. I'll do it sometime tonight.